Cargo.toml

```diff
@@ -50,18 +42,17 @@ num-traits = "0.2"
 pretty_assertions = "0.6"
 proj = { version = "0.14", optional = true }
 quick-xml = "0.17"
+relational_types = { version = "1", path = "../relational_types/" }
```

Comment:
```suggestion
relational_types = "1"
```

Reply:
Oops, that was for testing :smile: Fixed.